src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -88,12 +99,16 @@ type Props = {
 };
 
 type State = {
+    activeEntry?: FeedItem | null,
     isInputOpen: boolean,
+    isReplyDataLoading: boolean,
 };
 
 class ActivityFeed extends React.Component<Props, State> {
     state = {
+        activeEntry: undefined,
```

Comment:
We should avoid storing derived state from props in this way, since it can lead to synchronization issues. Let's retrieve the entry as needed using the value from `props.activeFeedEntryId`.